### PR TITLE
Feature: Work - About

### PR DESCRIPTION
**Pull request by luftedar**
### My Portfolio Project Header and Main

In this update:

-  Updated HTML Semantic Tags for Work and About Section.
-  Updated CSS for Work and About Sections. Grids and Flexboxes are Used.
-  Updated README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +41,128 @@ <h1>Hey there. I’m Orçun<br>
...
             </div>
+
+        </section>
+        <section id="card-grid">
+            <section id="projects">
+                <div id="projects-div">
+                    <h2 class="section-header">My Recent <br> Works</h2>
+                    <div id="green-line"></div>
+                </div>
+                <div id="projects-cards">
```
Comment (barackm) [inline index.html]: - [x] Nice job using `<section>` tags for the sections of your webpage, However, there is a couple of things we will need to fix 😉 . According to the project requirement, we are supposed to have two `<section>` tags, one for the ***Works*** section and the other for the **About myself** section, and they should be independent, but for now, I can see that you have multiple `<section>` tags and the **About myself** section is also inside the **Works** section, Kindly let's only have for now two `<section>` tags for the Works and About myself, and remember the about myself section should not be inside the Works section. 👍 


- [x] Good idea using the `display: grid`, but you did not use it at the right place, remember we need to use the `display: grid` to align the projects, see screenshot below 👇 
image, Kindly let's have the `display: grid` on the container of the projects 😉

File: index.html [hunk 2]
```diff
@@ -41,6 +41,128 @@ <h1>Hey there. I’m Orçun<br>
...
+                            </div>
+                            <div class="card-lang">
+                                <ul class="card-list">
+                                    <li class="card-list-element">Ruby on Rails</li>
+                                    <li class="card-list-element">CSS</li>
+                                    <li class="card-list-element">JavaScript</li>
+                                    <li class="card-list-element">HTML</li>
+                                </ul>
+                            </div>
+                            <div><button class="card-button">See Project</button></div>
```
Comment (barackm) [inline index.html]: - [x] Let us please have button interactions like Active, Hover, and Pressed on all the buttons, see the design below 👇 
image

File: style.css [hunk 3]
```diff
@@ -11,26 +15,43 @@ header {
...
+  background-size: contain;
   padding: 0 1.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 700px;
 }
 
 hgroup {
   display: flex;
```
Comment (barackm) [inline style.css]: - [x] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @luftedar, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

## To Highlight
✔️  Nice PR title
✔️  Nice PR summary
✔️  Good use of GitHub flow 👏 

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (luftedar) [reply index.html]: Hello. I want to be sure about this one. Is it enough to use :hover(for hover) and :active(for pressed)? Is there anything else for enabled? Should I also use the :visited?

Comment (barackm) [reply index.html]: Hi @luftedar, you can add other interactions if you wish, but the ones which are mandatory are Active, Hover, and Presse. and remember enabled is the default one so you don't need to specify it. Yes, you can use `:active` for pressed 😉

Comment (luftedar) [reply index.html]: Thanks @barackm you helped me a lot.  :100:

Comment (barackm) [reply index.html]: You're welcome @luftedar, Keep up the good work 😉

**Comment by luftedar** [comment]
###My Portfolio Project Header and Main:

In this update for requirements:

- Fixed Structure of Semantic Tags.
- Updated button hover and pressed effects.
- Updated cards' parent element to grid.
- Updated cards elements to flex-box.
- Applied DRY coding rules.

File: style.css [hunk 4]
```diff
@@ -1,39 +1,80 @@
...
 }
 
 header {
   display: flex;
+  flex-direction: column;
+  background: #fff;
+}
+
+nav {
+  display: flex;
```
Comment (TedLivist) [inline style.css]: - [x] In following with the DRY principle introduced by the previous reviewer, you did well by extracting the `display: flex` and `flex-direction: column` properties. But you seem to be using it anyways in these stylings. This defeats the purpose of the extraction. Please remove those properties in the elements that have them defined in and add them as class attributes in the HTML tags where appropriate.

For example, header should be
```
header {
  background: #fff;
}
```
and HTML should be
```
<header class="d-flex d-flex-column">
```

File: index.html [hunk 5]
```diff
@@ -42,6 +41,132 @@ <h1>Hey there. I’m Orçun<br>
...
+                                    <li class="card-list-element">CSS</li>
+                                    <li class="card-list-element">JavaScript</li>
+                                    <li class="card-list-element">HTML</li>
+                                </ul>
+                            </div>
+                            <div><button class="card-button g-btn">See Project</button></div>
+                        </div>
+                    </div>
+                    
+
```
Comment (TedLivist) [inline index.html]: - [x] As defined in the [HTML & CSS best practices link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):

blank_m

Please remove this spacing. Removing it makes your code more readable.

File: README.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ In this project, semantic HTML5 tags used while positioning for CSS made by usin
 
 ## Live Demo
 
-[Live Demo Link](https://raw.githack.com/luftedar/My-Portoflio/feature/index.html)
+[Live Demo Link](https://raw.githack.com/luftedar/My-Portoflio/feature-work-about/index.html)
```
Comment (TedLivist) [inline README.md:18]: - [ ] There seems to be something wrong with the display of your website and live link which was not present during the previous review. Please fix it. I have added more descriptions below.

Your project before the first review was just fine and scrollable. I can scroll down to the part of the page you can see in the image:

old_scr

But your project now is not scrollable at all. I have checked it on mobile and desktop. I can only see this part of the page by default: 

new_scr

and I need to zoom out before I can start seeing the `Recent Works` title. Even at that, I still cannot scroll to the bottom of the page like in the previous image.

**Review by TedLivist — CHANGES_REQUESTED** [review]
Hi @luftedar,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 6 above)
Comment (luftedar) [reply README.md:18]: Thanks for your review. My last push is not shown here but it was successful in my terminal. :disappointed: . I need to practice more for the git.

**Comment by luftedar** [comment]
###My Portfolio Project Header and Main:

In this update for requirements:

 -   Fixed Structure of Semantic Tags.
 -   Updated button hover and pressed effects.
 -   Updated cards' parent element to grid.
 -   Updated cards elements to flex-box.
 -   Applied DRY coding rules.

**Review by rukundoeric — APPROVED** [review]
Hi @luftedar,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 6 above)
Comment (TedLivist) [reply README.md:18]: Alright. Sure, over time you'll get the hang of it.

**Pull request merged**